src/map/skill.c

```diff
@@ -19717,8 +19717,16 @@ void skill_cooldown_load(struct map_session_data * sd)
 
 	// process each individual cooldown associated with the character
 	for( i = 0; i < cd->cursor; i++ ) {
-		cd->entry[i]->started = now;
-		cd->entry[i]->timer   = timer->add(timer->gettick()+cd->entry[i]->duration,skill->blockpc_end,sd->bl.id,cd->entry[i]->skidx);
+		int64 remaining;
+
+		if (battle_config.guild_skill_relog_delay == 2 && cd->entry[i]->skill_id >= GD_SKILLBASE && cd->entry[i]->skill_id - GD_SKILLBASE < MAX_GUILDSKILL) {
+			remaining = cd->entry[i]->started + cd->entry[i]->total - now;
```

Comment:
for clients older than 20120604 ``->total`` will be undefined or 0 depend on how enctry was created.
need remove existing ifdefs with ``->total``. You can search for 20120604 and see where this field hidden